packages/quantic/force-app/main/default/lwc/quanticCitation/quanticCitation.css

```diff
@@ -0,0 +1,52 @@
+.citation:hover {
+  cursor: pointer;
+  color: var(--lwc-brandAccessible, #0176d3);
+}
+
+.citation__badge {
+  max-width: 10.625rem;
+  height: 2rem;
+}
+
+.citation__badge:hover {
+  background-color: var(--lwc-brandLight, #f4f6fe);
+}
+
+.citation__badge:focus {
+  border: 1px solid var(--lwc-brandAccessible, #0070d2);
+  box-shadow: 0px 0px 3px 0px var(--lwc-brandAccessible, #0070d2);
+}
+
+.citation__badge:hover .citation__index {
+  background-color: var(--lwc-colorBackgroundAlt, #ffffff);
+}
+
+.citation__badge:hover .citation__title {
+  color: var(--lwc-brandAccessible, #0176d3);
+}
+
+.citation__index {
+  color: var(--lwc-brandAccessible, #0176d3);
+  background-color: var(--lwc-brandLight, #f4f6fe);
+  width: 20px;
+  height: 20px;
+  border-radius: var(--lwc-borderRadiusCircle, 50%);
+}
+
+.citation__title {
+  font-weight: var(--lwc-fontWeightRegular, 400);
+  color: var(--lwc-colorTextPlaceholder, #747474);
+  font-size: 13px;
```

Comment:
Shouldn't this font-size be coming from related font-size from somewhere else?

Xem or Xrem?